cypress/integration/internal/billing-cancel-bill-run.spec.js

```diff
@@ -74,6 +74,9 @@ describe('Cancel bill run', () => {
       cy.get('.govuk-radios__item').last().children().first().click()
       cy.get('button.govuk-button').click()
 
+      cy.get('#select-financial-year').click();
```

Comment:
We use the Defra standard linter now which does not like semi-colons

```suggestion
      cy.get('#select-financial-year').click()
```